### ISSUE TITLE
Provide an option to clear data folder during uninstalling

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#914]
Carried over from https://github.com/brave/browser-laptop/issues/2848
> After uninstalling Brave its data still remain in `AppData\Roaming\brave` on Windows. I think some users (especially privacy-oriented users) do not welcome this.

> Deleting profile info is not a typical uninstall behavior on any of the platforms. I don't think this should be the default because it'll surprise users. I think it makes sense to provide it as an option on two of the major platforms:
> - on Windows, the uninstall wizard can offer a checkbox (default: unchecked) to remove all Brave user data
> - on Linux `apt purge` (or equivalent) should remove profile info `apt remove` (/c.) should not

**Comment by bsclifton** [comment]
Chrome offers a remove profile directory when uninstalling- this may already be fixed on Windows

**Comment by srirambv** [comment]
@bsclifton yes this is an optional checkbox on uninstall prompt which removes the profile directory as well
![image](https://user-images.githubusercontent.com/17010094/44946421-1b6e1b80-adca-11e8-80cc-918e8d2f21fa.png)

On Linux there is no option to remove. Not so sure about mac.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
